src/shadow-viewport.js

```diff
@@ -133,8 +136,7 @@ ShadowViewport.prototype.processCTM = function() {
   this.originalState.x = newCTM.e
   this.originalState.y = newCTM.f
 
-  // Update viewport CTM and cache zoom and pan
-  this.setCTM(newCTM);
```

Comment:
This line is only used by `init()`. So i moved it out and place it after calling `processCTM()`.